tests/html/Document/body.html

```diff
@@ -0,0 +1,89 @@
+<!doctype html>
+<html>
+<head>
+<title>Document#body</title>
+<script>
+  (window.customElements = window.customElements || {}).forcePolyfill = true;
+</script>
+<script src="../../../../es6-promise/dist/es6-promise.auto.min.js"></script>
+<script src="../../../../web-component-tester/browser.js"></script>
+<script src="../../../custom-elements.min.js"></script>
+<script>
+function generateLocalName() {
```

Comment:
This doesn't seem like it's used anywhere.

Reply:
True, I'm going to remove that.